packages/utils/src/graphql/entities.ts

```diff
@@ -311,15 +316,12 @@ export function setJsonObjectType(
     //check if field is also json
     const typeString = extractType(field.type);
     const isJsonType = jsonObjects.map((json) => json.name).includes(typeString);
+    const jsonObject = jsonObjects.find((object) => object.name === typeString);
+    assert(jsonObject, `Json object ${typeString} not found`);
```

Comment:
This is a change in behaviour, the `jsonInterface` property can now never be undefined. 

If it is required, then we can remove the `isJsonType` otherwise we should remove the assertion